extensions/sql-database-projects/src/tools/netcoreTool.ts

```diff
@@ -78,8 +78,8 @@ export class NetCoreTool extends ShellExecutionHelper {
 			await vscode.commands.executeCommand('workbench.action.openGlobalSettings');
 		} else if (result === Install) {
 			//open install link
-			const dotnetcoreURL = 'https://dotnet.microsoft.com/download/dotnet-core/3.1';
-			await vscode.env.openExternal(vscode.Uri.parse(dotnetcoreURL));
+			const dotnetSdkUrl = 'https://dotnet.microsoft.com/download/dotnet/6.0';
```

Comment:
Should we create an aka.ms/somethingaboutsqlprojdotnet to put here such that we can recommend updated links without shipping a new version?

Reply:
sure good idea! Can you help with making one please :)

Reply:
https://aka.ms/sqlprojects-dotnet
:)